plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -138,6 +142,8 @@
     @Inject
     private NetworkModel _netMgr;
     @Inject
+    private HostDao _hostDao;
```

Comment:
Hi @sureshanaparti could you consider making a Jira? As a suggestion, could you remove those underscore "_" before variables? That convention is not recommended in Java.
Furthermore, could make a JavaDoc? That could save some time for those that are going to read your code in the future and turns things easier and organized.